docs/android/v2/how-to-guides/extend-capabilities/plugins/blur-filter.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Blur Filter (Beta)
+nav: 14.3
+---
+
+import AndroidSdkVersionShield from '@/common/android-sdk-version-shield.md';
+
+## Introduction
+
+Blur filter plugin helps in bluring the non-face background of a video frame.
```

Comment:
`Blur filter plugin helps in bluring the background of a video frame` 

Remove non-face from it .. Background by default means that